.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,219 @@
+          ALL_CHECKS=${CONFIG_CHECKS:-$DEFAULT_CHECKS}
+          JQ_SCRIPT=${JQ_SCRIPT_TEMPLATE/TOOLS/$ALL_CHECKS}
+          CHECKS=$(jq -c $JQ_SCRIPT < composer.json)
+          echo "Discovered checks: ${CHECKS}"
+          echo "::set-output name=check::${CHECKS}"
+
+  supported-versions-matrix:
+    name: Supported Versions Matrix
+    runs-on: ubuntu-latest
+    needs: [lint-json]
+    outputs:
+      version: ${{ steps.supported-versions-matrix.outputs.version }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: supported-versions-matrix
+        uses: WyriHaximus/github-action-composer-php-versions-in-range@v1
+
+  dependency-versions-matrix:
+    name: Dependencies to Test Matrix
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.dependency-versions-matrix.outputs.check }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: dependency-versions-matrix
+        run: |
+          DEPS=$(if [ -f composer.lock ]; then echo -n '["lowest", "locked", "latest"]'; else echo -n '["lowest", "latest"]'; fi)
+          echo "Testing against dependencies: ${DEPS}"
+          echo "::set-output name=check::${DEPS}"
+
+  stable-php:
+    name: Determine Stable PHP Version for Non-Unit Test Checks
+    runs-on: ubuntu-latest
+    needs: [supported-versions-matrix]
+    outputs:
+      version: ${{ steps.stable-php.outputs.version }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: stable-php
+        run: |
+          # This grabs the default stable PHP version for non-unit-test checks from the laminas.dev API.
+          # If the .github/workflows/continuous-integration-config.json file is
+          # present, and defines a string "stable-php" element, that value will
+          # be used.
+          DEFAULT_VERSION=$(curl -s https://laminas.dev/api/ci/stable-checks-version | jq -r '.version')
+          VERSION=$(if [ -f .github/workflows/continuous-integration-config.json ]; then jq -r 'def isstring(x): if x|type=="string" then x else empty end ; isstring(.["stable-php"])' < .github/workflows/continous-integration-config.json ; fi)
+          VERSION=${VERSION:-$DEFAULT_VERSION}
+          echo "PHP version discovered for non-unit-test checks: ${VERSION}"
+          echo "::set-output name=version::${VERSION}"
+
+  unit-tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    needs: [supported-versions-matrix, dependency-versions-matrix]
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ${{ fromJson(needs.supported-versions-matrix.outputs.version) }}
+        deps-version: ${{ fromJson(needs.dependency-versions-matrix.outputs.version) }}
+        include:
+          - php-version: "8.0"
+            composer-args: "--ansi --no-interaction --no-progress --prefer-dist --ignore-platform-reqs"
```

Comment:
As we are using composer v2 in CI, we can use `--ignore-platform-req=php` here, right?

Reply:
Good catch - fixed!